data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
+			choice
+				`	"Captain <last>."`
+
+			`	"Captain," she says, her eyes lighting up. "Well, as you may know, this is the last day of the exhibition. The captain I hired to transport everything back to <destination> is not only rude, but is now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60,000 credits, but I can pay you <payment> to take his contract instead. What do you say?"`
+			choice
+				`	"I would love to."`
+				`	"Sorry, I have more pressing matters at hand."`
+					goto decline
+
+			`	"Great!" Mariel says. "Bring your ship to this hangar, and I will have our crew get everything ready to load."`
+			`	After a couple hours, the whole exhibition is packed up and ready for transport to <planet>.`
+				accept
+
+			label decline
+			`	"Oh. I'm surprised you would turn down so much money for such little work as well as a chance to meet the artists. But it's your call." Mariel glances at her watch. "Well, anyway, the exhibition has closed. I'm going to have to ask you to leave."`
+				decline
+
+			label exit
+			`	Following where the woman went, you find the true exit of the warehouse and go back out onto the more familiar grounds of <origin>'s spaceport.`
+				decline
+
+	on accept
+		log `Saw an impressive exhibition on the Evacuation of Hope and ended up with a job taking the whole thing to Moonshake.`
+
+	on visit
+		dialog `You arrive on <planet>, but realize that your escort carrying the exhibition installations has not arrived yet. Better depart and wait for your escorts to enter the system.`
+
+	on complete
+		payment 80000
+		event "adelita 2 offer expired" 1
+		conversation
+			`After navigating the thick fog of Moonshake's atmosphere, you find a motley crew waiting on your landing pad. Once you sign your landing documents with the Syndicate officials, one of the crew walks up to you and hands you a credit chip for <payment> and a note. It reads:`
+			``
+			`Meet me at the Common Grounds Cafe in the spaceport if you're interested in assisting us with a new project. Make sure you're not followed.`
+			`	Adelita Zero`
+
+
+
+event "adelita 2 offer expired"
+
+
+
+mission "Adelita 2"
+	name "Film Syndicate sweatshop on <planet>"
+	description "Participate in an unauthorized filming of a Syndicate sweatshop on <destination>. (You will be paid <payment> after you complete the entire job.)"
+	cargo "stealth drones" 1
+	passengers 3
+	"apparent payment" 200000
+	source "Moonshake"
+	destination "Maker"
+	to offer
+		has "Adelita 1: done"
+		not "adelita 2 offer expired"
+
+	on offer
+		conversation
+			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat.`
```

Comment:
```suggestion
			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>, but after finding an information screen, you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat.`
```

Reply:
One too many commas for me, I think it breaks the flow.

Reply:
```suggestion
			`It takes a while to navigate through the bustling spaceport and the almost viscous fog of <origin>. After finding an information screen, you're able to get to the Common Grounds Cafe, which has a funky vibe to it and is quite busy. Nobody seems to recognize you, so you take a seat at a booth and order something to eat.`
```
How about this? I like the pause, but I agree that the sentence feels run-on-ish with just a comma thrown in. I personally think giving the longer pause with the period sounds better.